CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to Ubuntu Insights
+
+A big welcome and thank you for considering contributing to Ubuntu Insights and Ubuntu! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing this project. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Quicklinks
+
+- [Code of Conduct](#code-of-conduct)
+- [Getting Started](#getting-started)
+- [Issues](#issues)
+- [Pull Requests](#pull-requests)
+- [Contributing to the code](#contributing-to-the-code)
+- [Contributor License Agreement](#contributor-license-agreement)
+- [Getting Help](#getting-help)
+
+## Code of Conduct
+
+We take our community seriously and hold ourselves and other contributors to high standards of communication. By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://ubuntu.com/community/code-of-conduct).
+
+## Getting Started
+
+Contributions are made to this project via Issues and Pull Requests (PRs). A few general guidelines that cover both:
+
+- To report security vulnerabilities, please use the advisories page of the repository and not a public bug report. Please use [launchpad private bugs](https://bugs.launchpad.net/ubuntu/+source/Ubuntu-Insights/+filebug) which is monitored by our security team. On ubuntu machine, it’s best to use `ubuntu-bug Ubuntu Insights` to collect relevant information. FIXME: snap?
+- Search for existing Issues and PRs on this repository before creating your own.
+- We work hard to makes sure issues are handled in a timely manner but, depending on the impact, it could take a while to investigate the root cause. A friendly ping in the comment thread to the submitter or a contributor can help draw attention if your issue is blocking.
+- If you've never contributed before, see [this Ubuntu discourse post](https://discourse.ubuntu.com/t/contribute/26) for resources and tips on how to get started.
```

Comment:
The top of this discourse post is a redirect to https://ubuntu.com/community/contribute instead